ci/test_lightning.py

```diff
@@ -0,0 +1,24 @@
+from subprocess import call
+from pathlib import Path
+
+from .build import build_tests
+
+source_files = [
+    'integrations/pytorch-lightning/Neptune-PyTorch-Lightning-basic.ipynb',
+    'integrations/pytorch-lightning/Neptune-PyTorch-Lightning-advanced.ipynb'
+]
```

Comment:
@pitercl asked me to test Lightning integration only first. I'll include other notebooks in CI in another commit.

Reply:
ok